app/views/user_emails/_email_preview.erb

```diff
@@ -0,0 +1,20 @@
+<div class="list-group list-group-messages list-group-flush list-group-bordered">
```

Comment:
Lets use `ListGroupComponent` here. You can just add another arg called `messages` and follow the same pattern as the others. Lmk if you want to pair on this and knock it out.